plugin/pki/path_venafi_policy.go

```diff
@@ -217,9 +302,10 @@ func (b *backend) pathUpdateVenafiPolicy(ctx context.Context, req *logical.Reque
 	return &logical.Response{
 		Data: respData,
 	}, nil
+
 }
 
-func savePolicyEntry(policy *endpoint.Policy, name string, ctx context.Context, req *logical.Request) (policyEntry *venafiPolicyEntry, err error) {
+func savePolicyEntry(policy *endpoint.Policy, name string, ctx context.Context, storage logical.Storage) (policyEntry *venafiPolicyEntry, err error) {
```

Comment:
Are you sure? You replaced giving arguments to functions from link to copying.

Reply:
It's needed only for passing storage object